README.mdown

```diff
@@ -6,56 +6,110 @@ CakePHP Documentation
 
 This is the official documentation for the CakePHP project. It is available online in HTML, PDF and EPUB formats at http://book.cakephp.org.
 
-Contributing to the documentation is pretty simple. Please read the
-documentation on contributing to the documentation over on [the
-cookbook](http://book.cakephp.org/3.0/en/contributing/documentation.html) for
-help. You can read all of the documentation within as its just in plain text
-files, marked up with ReST text formatting.
+Contributing to the documentation is pretty simple. Please read the documentation on contributing to the documentation over on [the cookbook](http://book.cakephp.org/3.0/en/contributing/documentation.html) for help. You can read all of the documentation within as its just in plain text files, marked up with ReST text formatting.
 
-There are two ways for building the documentation: with Docker, or by installing
-the packages directly on your OS.
+There are three ways for using the documentation:
+- with Docker-compose : To build AND serve the documentation.
+- with Docker : Only if you need to build the documentation.
+- or by installing the packages directly on your OS.
 
-Build the Documentation with Docker
------------------------------------
+## Docker-compose : Build and serve the Documentation
 
-Docker will let you create a container with all packages needed to build the
-docs. You need to have docker installed, see the [official docs of
-docker](http://docs.docker.com/mac/started/) for more information.
+Docker-Compose will create and link multi-containers with all packages needed (including elasticsearch). You need to have [Docker](http://docs.docker.com/mac/started/) and [Docker-compose](https://docs.docker.com/compose/install/) installed.
 
-### Use the image on DockerHub ###
+### A shell to build and serve the docs
 
-The easiest way is to use the image already hosted on [DockerHub](https://hub.docker.com/r/cakephpfr/docs/).
+A `start.sh` script will install all the tools you need to build and serve the docs:
 
-You can directly run the following commands:
+    git clone git@github.com:cakephp-fr/docs-cakephp.git
```

Comment:
Not sure if we should be pointing people at the cakephp-fr repo. Could we put the required changes here?